### DISCUSSION TITLE
期待下一个版本

### DISCUSSION THREAD

**Discussion by duskroot** [kingToolbox/WindTerm#1286] [Ideas]
终于找到目前感觉最棒的ssh终端软件了，可以有一个很蛋疼的点，就是打开WindTerm时需要输入密码。而且锁屏时也需要，希望下一个版本能够取消这个功能！感谢！

**Comment by itInner** [comment]
input nothing  ,just  click  enter  
will set no password in open this program

**Comment by ViCrack** [comment]
现在就可以设置的

**Comment by loda13** [comment]
希望下一个版本尽快发，目前感觉是替代xshell最完美的终端
